zilliqa/tests/it/consensus.rs

```diff
@@ -219,6 +220,7 @@ async fn launch_shard(mut network: Network) {
 // and progresses.
 #[zilliqa_macros::test]
 async fn handle_forking_correctly(mut network: Network) {
+    trace!("Handle forking correctly test starting....");
```

Comment:
Should this trace be left in? I don't think the rest of our tests come with any logging by default outside of the network's message log.
On the other hand I guess it doesn't hurt, if you're debugging some other test you'll probably only run that one so this won't get in the way.

Reply:
I didn't intend on it being there, thanks :)

I don't think traces in the tests are so bad, though.